libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -344,18 +344,18 @@ proc validateAndRelay(g: GossipSub,
     # store in cache only after validation
     g.mcache.put(msgId, msg)
 
-    g.rewardDelivered(peer, msg.topicIds, true)
+    let topic = msg.topicId
+    g.rewardDelivered(peer, [topic], true)
```

Comment:
It might be interesting to change `rewardDelivered` to only take one topics instead of an openarray. Since it's used only on this file.

Reply:
I did think on changing this, but for some reason didn't check whether it was called somewhere else and then I forgot 🤦‍♀️

If you guys think this is not going to be called with multiple topics then I'm all for updating it. Do confirm, please :)

Reply:
+1 to change